src/Microsoft.Health.Abstractions/Properties/AssemblyInfo.cs

```diff
@@ -3,6 +3,8 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System;
 using System.Resources;
 
+[assembly: CLSCompliant(false)]
```

Comment:
Each of the assemblies is marked to indicate it does ensure compliance with the Common Language Specification (CLS)